packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/resolution/ResolutionFeature.java

```diff
@@ -129,7 +139,7 @@ public void setValue(ResolutionPreset value) {
   // Always supported
```

Comment:
This is no longer true with the latest change.

Reply:
Done.